plugins/lua/sort/places.lua

```diff
@@ -28,10 +28,22 @@ local zone_names = {
     [df.civzone_type.ClayCollection] = 'Clay'
 }
 
+-- I used strings rather than df.civzone_type because nobody is going to search "MeadHall" they're going to search "Tavern"
+local language_name_type_strings = {
+    [df.language_name_type.SymbolFood] = 'Inn/Tavern',
+    [df.language_name_type.SymbolDomestic] = 'MarketStall'
+}
+
 local function get_default_zone_name(zone_type)
     return zone_names[zone_type] or ''
 end
 
+local function get_location_type_from_language_name(language_name_type)
+    -- Looking at df.language.xml I think SymbolFood and SymbolDomestic are the only two we need to override?
+    -- Every other df.language_name_type for a location matches the displayed type name (e.g. Temples are language_name_type.Temple, etc)
```

Comment:
The ones that we care about are `Guildhall`, `Tavern`, `Temple`, and `Hospital`. The language name type seems to match up for everything except taverns, so you can create a static mapping for everything or just special case the taverns. Where does SymbolDomestic come up? I don't think I've seen that one in use.

Reply:
SymbolDomestic is for MarketStall -- I added it because both it and SymbolFood are written in df.language.xml as
`<enum-item name='SymbolDomestic' comment='MarketStall'>`

Looking through the wiki I don't think that's actually used anywhere (maybe Adventure Mode?) so I'll update it.
Hopefully final commit incoming ASAP! :)